app/assets/stylesheets/searchworks4/alerts.css

```diff
@@ -16,3 +16,12 @@
     margin-left: 0;
   }
 }
+
+.alert {
```

Comment:
Alerts are styled in the component-library already. We shouldn't need this repeated.

Reply:
removed the repetition